examples/basic.js

```diff
@@ -74,6 +74,28 @@ const Demo = React.createClass({
           </TreeNode>
         </TreeNode>
       </Tree>
+
+      <h2>Check on Click TreeNode</h2>
+      <Tree
+        className="myCls"
+        showLine
+        checkable
+        selectable={ false }
+        defaultExpandAll
+        defaultExpandedKeys={this.state.defaultExpandedKeys}
```

Comment:
Actually, you don't need to set `defaultExpandAll` and `defaultExpandedKeys` at the same time..

Reply:
lol, I just copied this from the 'simple' example